docs/installation/config-ksql.rst

```diff
@@ -3,77 +3,150 @@
 Configuring KSQL
 ================
 
-You can set configuration properties for KSQL and your queries with the
-SET statement. This includes :cp-javadoc:`settings for Kafka’s Streams
-API |streams/javadocs/index.html` (e.g.,
-``cache.max.bytes.buffering``) as well as settings for Kafka’s :cp-javadoc:`producer
-client |clients/javadocs/org/apache/kafka/clients/producer/ProducerConfig.html` and
-:cp-javadoc:`consumer
-client |clients/javadocs/org/apache/kafka/clients/consumer/ConsumerConfig.html`
-(e.g., ``auto.offset.reset``).
+You can set the default properties for KSQL, :cp-javadoc:`Kafka’s Streams |streams/javadocs/index.html`, Kafka’s
+:cp-javadoc:`producer client |clients/javadocs/org/apache/kafka/clients/producer/ProducerConfig.html` and
+:cp-javadoc:`consumer client |clients/javadocs/org/apache/kafka/clients/consumer/ConsumerConfig.html`, and admin client properties in the ``/etc/ksql/ksql-server.properties`` file.
 
-The basic syntax is:
+You can set per-session properties for :cp-javadoc:`Kafka’s Streams |streams/javadocs/index.html`, Kafka’s
+:cp-javadoc:`producer client |clients/javadocs/org/apache/kafka/clients/producer/ProducerConfig.html` and
+:cp-javadoc:`consumer client |clients/javadocs/org/apache/kafka/clients/consumer/ConsumerConfig.html`, and admin client in
+the SET statement in the CLI.
+
+To view the current configuration settings, use the ``SHOW PROPERTIES`` KSQL command.
 
 .. code:: sql
 
-    SET '<property-name>'='<property-value>';
+        ksql> SHOW PROPERTIES;
+
+Setting Default Properties
```

Comment:
This doesn't match our offline discussion on the Configuration section. For example, we want to clearly separate KSQL server configuration from KSQL CLI configuration.

Reply:
Filed this to work on immediately after merging: https://confluentinc.atlassian.net/browse/DOCS-344